pynear/include/crc32.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <cstddef>
```

Comment:
@Dobatymo this is very simple crc32 code I have found. Im not sure if it is efficient, but since its quite simple to use, Im using now. Let me know if you have suggestion of some other implementation. Thanks.